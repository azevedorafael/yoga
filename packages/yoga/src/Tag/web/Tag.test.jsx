import React from 'react';
import { render } from '@testing-library/react';
import { ThemeProvider, Tag } from '../..';

describe('<Tag />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider>
        <Tag />
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
