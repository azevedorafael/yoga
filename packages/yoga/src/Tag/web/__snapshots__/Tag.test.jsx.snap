// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> should match snapshot 1`] = `
.c0 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px  8px  8px  8px;
  background-color: #F5F5FA;
  color: #6B6B78;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c0 svg {
  margin-right: 5px;
}

<div>
  <div
    class="c0"
    color="#F5F5FA,#6B6B78"
  >
    <svg
      fill="#6B6B78"
      height="12"
      width="14"
    />
    default
  </div>
</div>
`;

exports[`<Tag /> should match snapshot with custom icon 1`] = `
.c0 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px  8px  8px  8px;
  background-color: #F5F5FA;
  color: #6B6B78;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c0 svg {
  margin-right: 5px;
}

<div>
  <div
    class="c0"
    color="#F5F5FA,#6B6B78"
  >
    <svg
      fill="#6B6B78"
      height="12"
      width="14"
    />
    default with custom icon
  </div>
</div>
`;

exports[`<Tag /> should match snapshot with full prop 1`] = `
.c0 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 8px  8px  8px  8px;
  background-color: #F5F5FA;
  color: #6B6B78;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c0 svg {
  margin-right: 5px;
}

<div>
  <div
    class="c0"
    color="#F5F5FA,#6B6B78"
  >
    <svg
      fill="#6B6B78"
      height="12"
      width="14"
    />
    full
  </div>
</div>
`;

exports[`<Tag /> should match snapshot with variant prop 1`] = `
.c0 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px  8px  8px  8px;
  background-color: #E9F3FC;
  color: #2186D9;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c0 svg {
  margin-right: 5px;
}

.c1 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px  8px  8px  8px;
  background-color: #DEF8E9;
  color: #1E854A;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c1 svg {
  margin-right: 5px;
}

.c2 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px  8px  8px  8px;
  background-color: #FCEAE9;
  color: #CB3530;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c2 svg {
  margin-right: 5px;
}

.c3 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px  8px  8px  8px;
  background-color: #FBFBBE;
  color: #CB9303;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c3 svg {
  margin-right: 5px;
}

<div>
  <div
    class="c0"
    color="#E9F3FC,#2186D9"
  >
    <svg
      fill="#2186D9"
      height="12"
      width="14"
    />
    informative
  </div>
  <div
    class="c1"
    color="#DEF8E9,#1E854A"
  >
    <svg
      fill="#1E854A"
      height="12"
      width="14"
    />
    positive
  </div>
  <div
    class="c2"
    color="#FCEAE9,#CB3530"
  >
    <svg
      fill="#CB3530"
      height="12"
      width="14"
    />
    negative
  </div>
  <div
    class="c3"
    color="#FBFBBE,#CB9303"
  >
    <svg
      fill="#CB9303"
      height="12"
      width="14"
    />
    warning
  </div>
</div>
`;

exports[`<Tag /> should match snapshot with without icon 1`] = `
.c0 {
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  padding: 8px  8px  8px  8px;
  background-color: #F5F5FA;
  color: #6B6B78;
  border-radius: 8px;
  font-size: 12px;
  font-weight: 700;
}

.c0 svg {
  margin-right: 5px;
}

<div>
  <div
    class="c0"
    color="#F5F5FA,#6B6B78"
  >
    default without icon
  </div>
</div>
`;
